packages/perseus/src/widgets/radio/__stories__/base-radio.stories.jsx

```diff
@@ -13,28 +13,63 @@ export default ({
     title: "Perseus/Widgets/Radio/Base Radio",
 }: Story);
 
-const defaultObject = {
+function generateChoice(options) {
+    const base = {
+        checked: false,
+        crossedOut: false,
+        content: "",
+        rationale: "",
+        hasRationale: false,
+        showRationale: false,
+        showCorrectness: false,
+        correct: false,
+        originalIndex: 0,
+        isNoneOfTheAbove: false,
+        highlighted: false,
+        previouslyAnswered: false,
+        revealNoneOfTheAbove: false,
+        disabled: false,
+    };
+
+    return {...base, ...options};
+}
+
+const defaultProps = {
```

Comment:
**suggestion (non-blocking):** The `__testdata__` folder may be a more appropriate for mock data (or functions that are used to generate mock data) that's shared between multiple files.

Reply:
Done 👍